platform/android/java/AndroidManifest.xml

```diff
@@ -25,12 +25,15 @@
 <!--The following values are replaced when Godot exports, modifying them here has no effect. Do these changes in the-->
 <!--export preset. Adding new ones is fine.-->
 
+        <!-- Metadata for VR app detection on Oculus devices. This is a no-op for regular Android apps, so it's safe-->
+        <!--to leave. -->
+        <meta-data android:name="com.samsung.android.vr.application.mode" android:value="vr_only"/>
```

Comment:
This we need to find out a way to make switchable. With this in the manifest we won't be able to deploy non VR games :)
Not sure how to do this because it means enhancements to the manifest parser which is some serious voodoo reverse engineering stuff that alters the binary version in the APK :)

Reply:
@BastiaanOlij That's not the case actually. Like I mentioned in the comment, that meta-data tag is a no-op on non-oculus devices, and on Oculus devices (i.e: Gear VR) it's also a no-op if the game doesn't make use of the Oculus SDK.
So it's perfectly safe to leave in the manifest.

This was tested and validated by exporting a regular Godot Android game with this patch and verifying that it runs as expected.

Reply:
@BastiaanOlij Actually, you might be correct. I had developer mode enabled on my Gear VR device which modified the expected behavior.

So after further testing, this is the observed behavior:
- On non-oculus devices the game runs and behaves as expected.
- On oculus devices (i.e: Gear VR) starting the game prompts to insert the device into a headset.

Thanks for catching that! I'll look into making it settable.